source/_integrations/oncue.markdown

```diff
@@ -30,3 +30,11 @@ The Oncue by Kohler integration will allow you to monitor the state of your [Onc
 - [KG80R](https://kohlerpower.com/en/residential/generators/product/kg80r)
 
 {% include integrations/config_flow.md %}
+
+## Non-functional and Incorrect Entities
+
+Some entities do not function with some generators.  For example the 20RESA does not report engine coolant temperature, and it will show as 0C (32F).  Also enot related are engine compartment temperature, current, and all related (power) entities.  On the 20RESA at least, the total generated power entry appears bogus.
+
+## Polling Considerations
+
+Currently the integration polls every 10 minutes.   Polling faster would have fresher data, and also use more resources.  Also there was at one point a report of someone being banned (unknown if permanent) for polling too often (unknown how fast).
```

Comment:
```suggestion
Currently, the integration polls every 10 minutes. Polling at a quicker interval may result in Kohler blocking your access to the API. Polling faster would have fresher data and also use more resources.
```

Reply:
I was thinking that we really do not know what's going on, other than that this uses 10 minutes and there are no bug reports about blocking.  It's true that quicker "may" lead to blocks, but that sounds like 5 is scary and more is known than actually is.  (I'm editing and will force-push.)